File: tree/main/src/events.py
# events.py
import random

def random_event():
    """
    Pilih dan tampilkan event random.
    Dipanggil misalnya setiap bladder % 30 == 0.
    """
    events = [
        "Kawan Sus 🤔 muncul!",
        "Elang Sigma 🦅 lewat!",
        "Toilet 🚽 terlihat di kejauhan...",
        "Toilet Monster 💀 mendekat!"
    ]
    event = random.choice(events)
    print(f"Event: {event}")
    return event

File: tree/main/src/events.js
// events.js
function triggerEvent(eventName) {
    switch(eventName) {
        case "teleport":
            console.log("✨ Event: Teleport sprite aktif!");
            break;
        case "overflow":
            console.log("💦 Event: Bladder overflow!");
            break;
        default:
            console.log(`📢 Event: ${eventName}`);
    }
}

module.exports = { triggerEvent };